corehq/apps/userreports/models.py

```diff
@@ -6,13 +6,13 @@
 from collections import namedtuple
 from copy import copy, deepcopy
 from datetime import datetime, timedelta
+from functools import cached_property
```

Comment:
I didn't realize this had made it into the standard library. Nice!

Reply:
oh cool, me neither!  Definitely feels like a good fit for it